src/main/java/com/example/fixba/controller/UserController.java

```diff
@@ -1,42 +1,55 @@
 package com.example.fixba.controller;
 
-import com.example.fixba.dto.UserDTO;
+import com.example.fixba.generated.model.UserContract;
+import com.example.fixba.generated.model.UserRegisterContract;
 import com.example.fixba.model.User;
 import com.example.fixba.service.UserService;
-import com.example.fixba.service.UserServiceImpl;
+import org.openapitools.api.UsersApi;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.http.ResponseEntity;
-import org.springframework.security.crypto.bcrypt.BCryptPasswordEncoder;
 import org.springframework.web.bind.annotation.*;
 
 import javax.validation.Valid;
+import javax.validation.constraints.Pattern;
 import java.util.List;
 
 @RestController
-@RequestMapping("/users")
-public class UserController {
-
-    UserService userService;
-
-    @Autowired
-    public UserController(UserService userService) {
-        this.userService = userService;
-    }
-
-    @GetMapping
-    public ResponseEntity<List<UserDTO>> getAll() {
-        return ResponseEntity.ok(userService.getAll());
-    }
-
-    @PostMapping
-    ResponseEntity<User> create(@Valid @RequestBody User user) {
-        return ResponseEntity.ok(userService.save(user));
-    }
-
-    @PostMapping("/login")
-    ResponseEntity<Boolean> logIn(@Valid @RequestBody User user) {
-        BCryptPasswordEncoder bCryptPasswordEncoder = new BCryptPasswordEncoder();
-        return ResponseEntity.ok(bCryptPasswordEncoder.matches(user.getPassword(), "$2a$10$xtL.q2MfuZCm0N6JjkTnmeM5TMAov10Vrzs15VlKJVWefib0mmFWW"));
-    }
-
+public class UserController implements UsersApi {
+
+	UserService userService;
+
+	@Autowired
+	public UserController(UserService userService) {
+		this.userService = userService;
+	}
+
+	@Override
+	public ResponseEntity<List<UserContract>> getUsers(@Valid Integer page, @Valid Integer size, @Valid String searchTerm, @Pattern(regexp = "^(((dueDate|title|createdAt){1}:)((asc|desc){1}))(,{1}(((dueDate|title|createdAt)+:)((asc|desc){1})))*$") @Valid String order) {
```

Comment:
Extract this regex to static string on top of the controller. Generally you should extract all string values on top of the class

Reply:
this here is generated code from open api scheme, how should i do this?